packages/test/test-service-load/src/gcDataStores.ts

```diff
@@ -597,7 +574,10 @@ export class DataObjectNonCollab extends BaseDataObject implements IGCActivityOb
 	 * Retrieves the oldest unreferenced data object, references it and asks it to run.
 	 */
 	private async reviveDataObject(dataObjectDetails: IActivityObjectDetails): Promise<boolean> {
-		console.log(`########## Reviving data object [${dataObjectDetails.id}]`);
+		logEvent(this.logger, {
+			eventName: "DS++",
```

Comment:
I thought this would be creating a DataStore.  Maybe `DS*` or `DS^`?

Reply:
It does not create a new data store. It just references an existing one.